src/components/checkbox/checkbox-group.spec.js

```diff
@@ -35,10 +34,6 @@ function render(props, childProps, renderer = mount) {
 }
 
 describe("CheckboxGroup", () => {
-  it("renders as expected", () => {
-    expect(render({}, {}, TestRenderer.create)).toMatchSnapshot();
-  });
-
```

Comment:
Do we no longer need that snapshot? I think that you need at least to write a test that checks if the structure is rendered as expected.

Reply:
What value would that provide that Chromatic doesn't? We lost no test coverage by removing this as well